crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -129,7 +129,7 @@ pub fn transaction_request_to_typed(
             }))
         }
         // EIP4844
-        (Some(3), None, _, _, _, Some(_), Some(_), Some(sidecar), Some(to)) => {
+        (Some(3), None, _, _, _, Some(_), Some(_), Some(sidecar), to) => {
```

Comment:
we should refactor this to use alloy's `TransactionRequest` -> `TypedTransaction` conversion at some point